### PR TITLE
Updated playbook using as source the playbook in: https://github.com/SnceGroup/digital-ocean-lamp-playbook

### PR DESCRIPTION
**Pull request by sgentilini**
Compared to DO version:
- DO specific files are not included
- No users creation
- No system reboot at the end of the provisioning

Compared to previous vagrant version:
- removed Compass

**Please cross check the diff of include/apache.yml if commented folders creations are necessary**

### CODE REVIEW

File: vagrant/ansible/include/apache.yml [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
   yum: pkg=httpd state=latest
 - name: Ensure httpd is running automatically at boot time and start it
-  action: command /sbin/chkconfig httpd on
+  shell: systemctl enable httpd.service
 #eZ Publish standard VHost
 #- name: Html folders creations
   #file: path={{ item.doc_root }} state=directory owner={{ item.user }} group={{ item.user }}
   #with_items:
-  #  - '{{ apache_vhosts_ez }}'
+    #- "{{ apache_vhosts_ez }}"
```
Comment (gabriperego) [inline vagrant/ansible/include/apache.yml]: @sgentilini did you already test this VHost configuration? I have some doubts that it works on Vagrant Shared Folders

**Comment by sgentilini** [comment]
Please look at the new commit

File: vagrant/ansible/include/apache.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
   yum: pkg=httpd state=latest
 - name: Ensure httpd is running automatically at boot time and start it
-  action: command /sbin/chkconfig httpd on
+  shell: systemctl enable httpd.service
 #eZ Publish standard VHost
-#- name: Html folders creations
-  #file: path={{ item.doc_root }} state=directory owner={{ item.user }} group={{ item.user }}
-  #with_items:
-  #  - '{{ apache_vhosts_ez }}'
+- name: Set proper ACL permission for apache user
```
Comment (gabriperego) [inline vagrant/ansible/include/apache.yml]: We can also remove this ACL

**Comment by sgentilini** [comment]
Updated based on last comment. Please look at the new commit

**Comment by gabriperego** [comment]
Merged!

**Pull request merged**